cluster/saltbase/salt/rescheduler/rescheduler.manifest

```diff
@@ -28,8 +28,8 @@ spec:
     # TODO: split this out into args when we no longer need to pipe stdout to a file #6428
     - sh
     - -c
-    - '/rescheduler --running-in-cluster=false 1>>/var/log/rescheduler.log 2>&1'
+    - '/rescheduler --listen-address=127.0.0.1:9235 --running-in-cluster=false --log_dir=/var/log/rescheduler --logtostderr=false'
```

Comment:
when use localhost here, it will need to resolve the hostname and may be failed if the dns have some issues.